integration-tests/test.sh

```diff
@@ -32,7 +32,8 @@ docker_exec() {
 if docker ps -a | grep "$container_name";then
   docker rm -vf "$container_name"
 fi
-opts=(--env OS=$os --cap-add SYS_ADMIN -v "$PWD/goss:/goss"  -d --name "$container_name" $(seccomp_opts))
+# We need --privileged to make systemd work.
```

Comment:
Why is that different from upstream?

Reply:
I'm not sure but maybe `centos:7.2.1511` changed their systemd setup somehow. In any case, `make test-int` does not work for me locally without this `--privileged` flag.